packages/jest-snapshot/src/__tests__/utils.test.js

```diff
@@ -198,3 +199,15 @@ test('serialize handles \\r\\n', () => {
 
   expect(serializedData).toBe('\n"<div>\n</div>"\n');
 });
+
+describe('DeepMerge', () => {
+  it('Correctly merges objects with property matchers', () => {
+    const target = {data: {bar: 'bar', foo: 'foo'}};
+    const matcher = expect.any(String);
+    const propertyMatchers = {data: {foo: matcher}};
+    const mergedOutput = deepMerge(target, propertyMatchers);
+
+    expect(mergedOutput).toStrictEqual({data: {bar: 'bar', foo: matcher}});
+    expect(target).toStrictEqual({data: {bar: 'bar', foo: 'foo'}});
```

Comment:
@swcisel I updated your tests here, does this make sense?

Reply:
Looks good to me. thanks!